src/components/content.js

```diff
@@ -444,13 +445,7 @@ class Content extends React.Component {
   onDragOver = (event) => {
     if (!this.isInEditor(event.target)) return
 
-    const { dataTransfer } = event.nativeEvent
-    const data = getTransferData(dataTransfer)
-
-    // Prevent default when nodes are dragged to allow dropping.
-    if (data.type == 'node') {
-      event.preventDefault()
-    }
+    event.preventDefault()
```

Comment:
It wasn't being checked anyway; it's not possible to access this information within ondragover. The DragEvent is in "protected mode" (one reason for this mode is to prevent data sniffing when you drag past other windows to get to the desired drop destination.)